manim/mobject/mobject.py

```diff
@@ -1385,6 +1397,7 @@ def is_off_screen(self):
     def stretch_about_point(self, factor, dim, point):
         return self.stretch(factor, dim, about_point=point)
 
+    @deprecated
```

Comment:
missing some arguments here:

`since`, `until` and `replacement`.

Reply:
missed out on that one 😅